packages/pouchdb-adapter-utils/src/preprocessAttachments.js

```diff
@@ -1,72 +1,81 @@
 import {
   atob,
   btoa,
```

Comment:
I took this opportunity to clean up `preprocessAttachments` and move a lot of functions-within-functions into their own functions, which is better for perf and easier to read.